docs/how-to/perform-site-failover.rst

```diff
@@ -0,0 +1,71 @@
+=============================================
+Perform failover for replicated RBD resources
+=============================================
+
+In case of a disaster, all replicated RBD pools can be failed over to a non-primary remote.
```

Comment:
What about failover of images?

Reply:
In the first implementation, only cluster (site) level failover/failback are supported.

Reply:
The images get promoted/demoted accordingly to their parent pools.